doc/cabal-package-description-file.rst

```diff
@@ -1992,6 +1992,13 @@ system-dependent values for these fields.
     arguments are compiler-dependent, this field is more useful with the
     setup described in the section on `system-dependent parameters`_.
 
+.. pkg-field:: jspp-options: token list
+
+    Command-line arguments for pre-processing Haskell code. Applies to
```

Comment:
This description looks copied from below, and incorrect. `jspp-options` should only be used for preprocessing .js files?

Reply:
Maybe it should be replaced with something like that?
```
    Command-line arguments for pre-processing JS code. Applies to pre-processed
    Haskell source like .js. Flags here will be passed as ``-optJSP`` flags to GHC.
```